common/flatpak-appdata.c

```diff
@@ -291,6 +368,8 @@ flatpak_parse_appdata (const char  *appdata_xml,
           *comments = g_hash_table_ref (component->comments);
           *version = g_steal_pointer (&component->version);
           *license = g_steal_pointer (&component->license);
+          *content_rating_type = g_steal_pointer (&component->content_rating_type);
```

Comment:
Some user below passes null for content_rating_type and content_rating, so you need to check it here.

Reply:
Fixed locally, although I explicitly missed out the check because it’s missing from all the other arguments in that function.

Reply:
Well, its only *really* needed in the initial commits, later commits i believe change all callers to non-NULL...

Reply:
> later commits i believe change all callers to non-NULL...

That is correct.

I’ll leave the `NULL` checks in. It’s a less surprising calling convention in any case.